app/components/ExternalLinksBlock/ExternalLinksBlock.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { ThemeProvider } from 'styled-components';
+import theme from 'styled-theming';
+import ExternalLink from './components/ExternalLink';
+import { zooTheme } from '../../theme';
+
+export const StyledExternalLinksBlock = styled.div`
+  background-color: white;
+  box-sizing: border-box;
+  color: ${theme('mode', {
+    light: zooTheme.colors.navy.default
+  })};
+  flex: ${props => `1 0 ${props.basis}%`};
+  min-width: 400px;
```

Comment:
What about using the `pxToRem()` function? [(example)](https://github.com/zooniverse/Panoptes-Front-End/blob/e07c34edad9d1ea5773781453ce4d3d689567ddc/app/pages/project/components/ProjectNavbar/components/ProjectNavbarNarrow/NarrowMenuButton.jsx)

Reply:
We should probably lay down some rules on when to use which unit as well...

Reply:
This is to enforce a specific layout, so pixels make sense to me here. Having some rules on fixed vs responsive unit usage would be useful.